src/browser/provider/plugin-host.js

```diff
@@ -0,0 +1,64 @@
+import { assignIn } from 'lodash';
+
+
+const helpers = Symbol();
+
+
+export default class BrowserProviderPluginHost {
+    constructor (providerObject, helpersObject) {
+        assignIn(this, providerObject);
+
+        this[helpers] = helpersObject;
```

Comment:
Do we need to store helpers in plugin? Can we just use helpers from the outside? So we will not have methods that can be augmented?